.pre-commit-config.yaml

```diff
@@ -59,3 +59,18 @@ repos:
           - -w      # autofix in place
           - -d      # show diff
           - -i=2    # indent 2 spaces
+
+  # Example usage of dotnet-format: pre-commit run dotnet-format --all-files
+  - repo: local
+    hooks:
+      - id: dotnet-format
+        name: dotnet-format
+        entry: script/check-dotnet-format.sh
+        language: script
+        types:
+          - file
+          - text
+        pass_filenames: false
+        files: ^.*/.*.cs$
+        args:
+          - ./samples/
```

Comment:
Is this intentional? If so, it might be better to introduce some structure, like `samples/{hook-name-that-these-are-samples-for}`, so we can understand more about which samples are for which hook.

Reply:
I'll make that change now